src/abstracts/SablierMerkleBase.sol

```diff
@@ -59,10 +60,10 @@ abstract contract SablierMerkleBase is
     //////////////////////////////////////////////////////////////////////////*/
 
     /// @notice Constructs the contract by initializing the immutable state variables.
-    constructor(MerkleBase.ConstructorParams memory params, uint256 fee) Adminable(params.initialAdmin) {
+    constructor(address campaignCreator, MerkleBase.ConstructorParams memory params) Adminable(params.initialAdmin) {
```

Comment:
Can we please switch the order of these parameters so that `baseParams` comes first?